packages/cosmos_lint/lib/lint.dart

```diff
@@ -0,0 +1,14 @@
+/// The lint package doesn't ship any dart source code.
+///
+/// To enable `lint`,
+/// 1. Add it to your dev_dependencies
+/// ```yaml
+/// dev_dependencies:
+///   lint: ^1.0.0
+/// ```
+///
+/// 2. Include the rules into your `analysis_options.yaml`
+/// ```yaml
+/// include: package:lint/analysis_options.yaml
+/// ```
```

Comment:
this documentation is copied from `lint` package and is misleading, either remove it or update it with our `cosmos_lint` package specifics

Reply:
Sure

Reply:
replacing it